docs/customization/customizing-emails.mdx

```diff
@@ -0,0 +1,174 @@
+
+      # Here you can add your custom code to calculate the ETA of the package:
+      @eta = DateTime.now + 1.day
+
+      options = { resend: false }.merge(options)
+      @store = @order.store
+      subject = (options[:resend] ? "[#{t('spree.resend').upcase}] " : '')
+      subject += "#{@store.name} #{t('spree.shipment_mailer.shipped_email.subject')} ##{@order.number}"
+      mail(to: @order.email, from: from_address(@store), subject: subject)
+    end
+  end
+end
+```
+
+Now we need to configure Solidus to use this new mailer instead of the default one:
+
+```ruby title="config/initializers/spree.rb"
+
+Spree.config do |config|
+  config.carton_shipped_email_class = 'AmazingStore::CartonMailer'
+end
+```
+
+This is the list of all the extension points available in Solidus today:
+
+- `order_mailer_class` which defaults to [Spree::OrderMailer][OrderMailer]
+- `carton_shipped_email_class` which defaults to [Spree::CartonMailer][CartonMailer]
+- `reimbursement_mailer_class` which defaults to [Spree::ReimbursementMailer][ReimbursementMailer]
+- `promotion_code_batch_mailer_class` which defaults to [Spree::PromotionCodeBatchMailer][PromotionCodeBatchMailer]
+
+[OrderMailer]: https://github.com/solidusio/solidus/tree/master/core/app/mailers/spree/order_mailer.rb
+[CartonMailer]: https://github.com/solidusio/solidus/tree/master/core/app/mailers/spree/carton_mailer.rb
+[ReimbursementMailer]: https://github.com/solidusio/solidus/tree/master/core/app/mailers/spree/reimbursement_mailer.rb
+[PromotionCodeBatchMailer]: https://github.com/solidusio/solidus/tree/master/core/app/mailers/spre/promotion_code_batch_mailer.rb
+
+### Customize the email template
+
+Next step is adding this new information somewhere in the actual email template, which
+represents what your customers will see in their inbox. The default templates for the
+original carton mailers in Solidus are located at
+[`core/app/views/spree/carton_mailer/`](https://github.com/solidusio/solidus/tree/935c8aeaaa2e427b6ea8f53d220b99aef2d98bf3/core/app/views/spree/carton_mailer).
+
+:::info
+
+Email templates are made of two main parts:
+
+1. A layout, which is identical in all emails
+2. A content, which is different based on the content.
+
+In this example, we are going to customize the latter, but if you need to make changes to the
+layout, you can just copy/paste
+[the one provided by Solidus](https://github.com/solidusio/solidus/blob/935c8aeaaa2e427b6ea8f53d220b99aef2d98bf3/core/app/views/layouts/spree/base_mailer.html.erb)
+in your own application and change it based on your style preferences.
+
+:::
+
+We can copy paste those files into our project at `app/views/awesome_store/carton_mailer/`.
+Now, rendering our ETA is a matter of adding something like this when we prefer within the
+`shipped_email.*.erb` templates:
+
+```erb
+Will be at your door by <%= @eta.to_s(:short) %>
+```
+
+At this point you probably want to see how your email looks like without creating a new order
```

Comment:
This paragraph is a bit convoluted :slightly_smiling_face:

Reply:
You are right, there's no reason to reinforce the benefit of this Rails feature for the user. They just need to get the information. I will simplify this radically.